README.md

```diff
@@ -101,47 +101,78 @@ Follow the steps below to set up a testnet smart wallet using `starkli`:
 
 ## Declare and Deploy Contracts in Testnet
 
+### Ethereum smart contract
+First, the Ethereum smart contracts must be deployed. For Ethereum the deployment process you will need to:
+
+1. Create your `.env` file: you need to configure the following variables in your own .env file on the contracts/solidity/ folder. You can use the env.example file as a template for creating your .env file, paying special attention to the formats provided
+
+   ```bash
+   ETH_RPC_URL = Infura or Alchemy RPC URL
+   ETH_PRIVATE_KEY = private key of your ETH wallet
+   ETHERSCAN_API_KEY = API Key to use etherscan to read the Ethereum blockchain
+   ```
+   **NOTE**:
+
+   - You can generate ETHERSCAN_API_KEY [following this steps](https://docs.etherscan.io/getting-started/creating-an-account).
+   - For the deploy, you will need some GoerliETH that you can get from this [faucet](https://goerlifaucet.com/).
+
+
+2. Deploy Solidity contract
+
+   ```bash
+      make ethereum-deploy
+   ```
+
+### Starknet smart contracts
+
+After the Ethereum smart contract is deployed, the Starknet smart contracts must be declared and deployed.
 On Starknet, the deployment process is in two steps:
 
 - Declaring the class of your contract, or sending your contract’s code to the
   network
 - Deploying a contract or creating an instance of the previously declared code
   with the necessary parameters
 
-1. Updated `.env` file: Please modify the variables with your Testnet account and your RPC provider.
+For this, you will need to:
+
```

Comment:
We should add here the addresses for the Herodotus Fact Registry for Goerli, Sepolia and Mainnet.
You can take a look at this [website](https://docs.herodotus.dev/herodotus-docs/developers/contract-addresses) and we are already asking Herodotus for the Sepolia addresses.

Reply:
Did you also ask for the Starknet Mainnet addresses? They have only published Starknet Goerli Addresses

Reply:
They just answered we can find them here:
 https://herodotus.notion.site/Contract-deployments-bfeacfc2d7b94c48aad688dbad553540